pkg/virt-controller/watch/snapshot/source.go

```diff
@@ -148,12 +159,78 @@ func (s *vmSnapshotSource) Unlock() (bool, error) {
 	return true, nil
 }
 
-func (s *vmSnapshotSource) Spec() snapshotv1.SourceSpec {
-	vmCpy := s.vm.DeepCopy()
-	vmCpy.Status = kubevirtv1.VirtualMachineStatus{}
+func (s *vmSnapshotSource) HaveHotplug() (bool, error) {
+	vmi, exists, err := s.controller.getVMI(s.vm)
+	if err != nil || !exists {
+		return false, err
+	}
+
+	vmRevision, err := s.GetVMRevision()
+	if err != nil {
+		return false, err
+	}
+
+	vmiPVCs := getPVCsFromVolumes(vmi.Spec.Volumes)
+	vmRevisionPVCs := getPVCsFromVolumes(vmRevision.Spec.Template.Spec.Volumes)
+
+	return !reflect.DeepEqual(vmiPVCs, vmRevisionPVCs), nil
```

Comment:
Is it possible to have the same elements but in different order in the lists?

Reply:
I thought deepequal checks for that kind of stuff

Reply:
Pretty sure `reflect.DeepEqual([1,2,3], [2,1,3]) == false`

Reply:
in this case we are comparing two dictionaries so the deepequal should be ok